test/functional/file-name-debuginfo/test.bats

```diff
@@ -29,10 +37,20 @@ setup() {
   [[ 1 -eq $(grep '/usr/bin/foobar$' $DIR/www/10/Manifest.test-bundle | wc -l) ]]
   [[ 1 -eq $(grep '/usr/lib/debug$' $DIR/www/10/Manifest.test-bundle | wc -l) ]]
   [[ 1 -eq $(grep '/usr/src/debug$' $DIR/www/10/Manifest.test-bundle | wc -l) ]]
-  # These should be pruned
-  [[ 0 -eq $(grep '/usr/src/debug/bar$' $DIR/www/10/Manifest.test-bundle | wc -l) ]]
-  [[ 0 -eq $(grep '/usr/lib/debug/foo$' $DIR/www/10/Manifest.test-bundle | wc -l) ]]
+  # These should be pruned right now, but should be pruned with the next update
```

Comment:
These should *not* be pruned right now, I think it what you mean.

Reply:
typo fixed.